src/main/kotlin/coverosR3z/misc/utility/GeneralUtilities.kt

```diff
@@ -70,13 +70,14 @@ fun checkParseToLong(value: String?,
 
 /**
  * Given a set of keys needed by an API (see files ending in "API") and the
- * actual data received, throw an error if it doesn't precisely match
+ * actual data received, throw an error if it doesn't have the required inputs
+ * (Note: there may be *more* inputs sent than this, but these are the required ones)
  * @param receivedKeys the data sent to the API
  * @param requiredKeys the exact keys required
  */
-fun checkHasExactInputs(receivedKeys: Set<String>, requiredKeys: Set<Element>) {
```

Comment:
relaxing our standards a bit.  Now, there are _required_ inputs, and optional inputs.  Before this change, it would require the exact set of inputs to arrive at the API, now it is allowable to have more inputs than these received.